frameworks/elastic/tests/test_duplicate_java_flags.py

```diff
@@ -1,7 +1,36 @@
 import pytest
-import retrying
-import sdk_install
-import sdk_networks
-import sdk_tasks
 from tests import config
 
+@pytest.fixture(scope="module")
+def elastic_service(service_account):
+    service_options = {
+        "service": {
+            "name": config.SERVICE_NAME,
+            "service_account": service_account["name"],
+            "service_account_secret": service_account["secret"],
+            "security": {"transport_encryption": {"enabled": True}},
+        },
+        "elasticsearch": {"xpack_enabled": True},
+    }
+
+    sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+    try:
+        sdk_install.install(
+            config.PACKAGE_NAME,
+            service_name=config.SERVICE_NAME,
+            expected_running_tasks=config.DEFAULT_TASK_COUNT,
+            additional_options=service_options,
+            timeout_seconds=30 * 60,
+        )
+
+        yield {**service_options, **{"package_name": config.PACKAGE_NAME}}
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+def test_duplicate_flags(elastic_service):
```

Comment:
This test could be moved to the `test_sanity` module, and the `test_duplicate_java_flags` module could be removed.

Reply:
Okay, Doing it.